trade-finance-manager-api/src/v1/helpers/activity.js

```diff
@@ -129,7 +129,7 @@ const getActivities = (deal) => {
  * Return array of activities object, comprising of
  * ACBS interaction records
  * @param {Object} acbs Durable function output
- * @returns {Array} An array of activities object
+ * @returns {Promise<object[]|{}>} An array of activities object
```

Comment:
This is an annoying return type, but I see why this is it. Either way, same formatting rules as above - `Promise<object[] | {}>`